js/src/pages/UserDetails.tsx

```diff
@@ -183,7 +183,6 @@ export const UserDetailsPage: React.FC = () => {
     <div className="human-resource-details">
       <Menu
         onClick={(e: any) => {
```

Comment:
you can get "key" after restructuring, onClick={({key}) => {setSelected(key);}}

Reply:
done